daemon/mgr/container_exec.go

```diff
@@ -27,18 +29,48 @@ func (mgr *ContainerManager) CreateExec(ctx context.Context, name string, config
 		return "", fmt.Errorf("container %s is not running", c.ID)
 	}
 
+	Envs := MergeEnvs(c.Config.Env, config.Env)
+	logrus.Info("Envs", Envs)
+
 	execid := randomid.Generate()
 	execConfig := &ContainerExecConfig{
 		ExecID:           execid,
 		ExecCreateConfig: *config,
 		ContainerID:      c.ID,
+		Env: 			 Envs,
 	}
 
 	mgr.ExecProcesses.Put(execid, execConfig)
 
 	return execid, nil
 }
 
+// MergeEnvs merge oldEnvs with new Env
+func MergeEnvs(oldEnv, newEnv []string) []string {
```

Comment:
I am wondering if that we have already supported this function, in https://github.com/alibaba/pouch/blob/master/daemon/mgr/container_utils.go#L262 
I think this is a common function.